### ISSUE TITLE
Missions – Ne pas pré-remplir le formulaire de contrôle avec des infos vieilles de plus de 24h après le LAN

### ISSUE THREAD

**Issue by AdelineCelier** [MTES-MCT/monitorfish#2043]
Vu pendant les tests de la fonctionnalité "pré-remplissage du formulaire de contrôle avec le JPE" : le formulaire peut être pré-rempli avec des infos assez vieilles. 
L'idée serait de dire que si le dernier message est un LAN et qu'il date de plus de 24h, on ne pré-remplit pas le formulaire avec les infos du JPE.

**Comment by VincentAntoine** [comment]
yes on peut faire ça pour commencer, et ça pose même la question d'être encore plus strict, de ne pré-remplir que si le dernier message (quel qu'il soit) a moins de 24 ou 48h, car en activité les navires sont censés a minima émettre un FAR par jour donc en l'absence de messages récents il est probable que les données ne soient pas à jour.
